cfme/utils/appliance/__init__.py

```diff
@@ -1681,16 +1688,20 @@ def is_embedded_ansible_running(self):
         supervisord = self.supervisord.running if self.version < '5.11' else True
         return self.is_embedded_ansible_role_enabled and supervisord
 
-    def wait_for_embedded_ansible(self, timeout=1200):
+    def wait_for_embedded_ansible(self):
         """Waits for embedded ansible to be ready
 
         Args:
             timeout: Number of seconds to wait until timeout (default ``1200``)
         """
+
+        timeout = 1200
```

Comment:
**Required** I don't think this value should be hard-coded. Why have you removed it as an argument?

Reply:
I removed it from the arguments because the value used to be mangled in this function in case the appliance is pod.

I guess you are suggesting to put this somwhere as a "constant" right? Perhaps on top of the module. Then do you think this function should even exist? I don't see much of value over just calling the wait_for instead of this.

Reply:
I think we should leave the argument in and, if is not specified then we should modify it. I also think the function should exist, as it's useful to have these convenience functions so you don't have to remember exactly what to `wait_for`. So what I'm suggesting is:

```python
default_timeout = 1200 == timeout
if self.is_pod and default_timeout:
    timeout *= 2
if self.version < '5.11' and default_timeout:
    timeout *=2
```
If a user passes in something other than the default timeout then we should use that.

Reply:
I did it bit differently.
I also decreased the timeout when we are dealing with 5.11 as there I think the ansible should be available almost instantly. What do you thinkg @sbulage ?

Reply:
Yes @jaryn on 5.11 version Embedded Ansible gets enabled instantly.